governance/FeatureDeprecation.md

```diff
@@ -0,0 +1,26 @@
+# Feature Deprecation
+
+# Overview
+
+This document aims to describe how to handle system/feature level deprecations that will impact customers.
+
+## Steps
+
+When looking to deprecate a feature from O3DE, please follow the run book below:
+
+1. Start the sig notification process ideally three releases prior to final deprecation:
```

Comment:
Again, need RFC for feature deprecation.

Reply:
Agreed :+1: